requirements/base.txt

```diff
@@ -0,0 +1 @@
+pyyaml
```

Comment:
Should we pin these requirements as well?

Reply:
We could, although there are more pyyaml releases (major, minor) than ubuntu or python. We could pin it to `ppyaml < 7`, so if a new breaking version is introduced, we'll need to update it manually. What do you think?

Reply:
We could only pin Python to 3.11, and this will work fine. But I also pinned Ubuntu to avoid this kind of failure in the near future.

Reply:
I agree with what you say, I think we can leave it as it is.